main.py

```diff
@@ -75,7 +75,7 @@ def search(query: str) -> Dict[str, Any]:
 
 
 @app.get("/extract")
-def extract(entity_json: str) -> Dict[str, Any]:
+def extract(entity_json: str) -> StreamingResponse:
```

Comment:
Good catch. Seems like pydantic doesn't really typecheck if it was allowed to be wrong in the first place?

Reply:
It only type checks when you run it with a linter. I used the `make format` & `make lint` the readme.